omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -203,7 +254,21 @@ public void setModelAttributes(PageModel model, Account account, OtherAccountDat
 		model.addAttribute("allowedLocales", administrationService.getAllowedLocales());
 		List<ProviderRole> providerRoles = providerManagementService.getAllProviderRoles(false);
 		model.addAttribute("providerRoles", providerRoles);
-		
+
+		List<Extension> customPersonEditFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountPersonFragment");
+		Collections.sort(customPersonEditFragments);
+		model.addAttribute("customPersonEditFragments", customPersonEditFragments);
+		List<Extension> customPersonViewFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountPersonViewFragment");
+		Collections.sort(customPersonViewFragments);
+		model.addAttribute("customPersonViewFragments", customPersonViewFragments);
+
+		List<Extension> customUserEditFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountUserFragment");
```

Comment:
If this is dealing with strictly user properties, then it should include "UserProperty" in its name.

Reply:
changed